proparse/src/main/java/org/prorefactor/treeparser01/TP01Support.java

```diff
@@ -1197,21 +1197,22 @@ private ITable astTableLink(AST tableAST) {
     return buffer.getTable();
   }
 
-  private SymbolScopeSuper classStateSuper(JPNode classNode, String className) throws RefactorException {
-    LOG.trace("Entering classStateSuper {} {}", classNode, className);
+  private TreeParserSuperSymbolScope classStateSuper(JPNode classNode, String className) throws RefactorException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "classNode". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1172)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.prorefactor.refactor.RefactorException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)